src/kernel/arch/x86/arch.zig

```diff
@@ -8,6 +8,7 @@ const irq = @import("irq.zig");
 const isr = @import("isr.zig");
 const pit = @import("pit.zig");
 const rtc = @import("rtc.zig");
+const x86_serial = @import("serial.zig");
```

Comment:
Does it need to be `x86_serial` as we are already in x86, so `serial` would be fine?

Reply:
No it certainly doesn't!